airbyte-cdk/java/airbyte-cdk/s3-destinations/src/main/kotlin/io/airbyte/cdk/integrations/destination/s3/BlobStorageOperations.kt

```diff
@@ -53,6 +53,22 @@ abstract class BlobStorageOperations protected constructor() {
         pathFormat: String
     )
 
+    /** Clean up all the objects matching the provided [keysToDelete] */
+    abstract fun cleanUpObjects(keysToDelete: List<String>)
```

Comment:
doublechecking: is this conceptually equivalent to `keysToDelete.forEach {blobStorageOp.dropBucketObject(it)}`?

Reply:
Probably so but the implementation was weird, that one uses param as list endpoint prefix. so we will be listing objects with 1 exact match for each key (instead of a group of keys)